src/autoscaler/cf/app.go

```diff
@@ -179,69 +193,24 @@ func (c *Client) get(url string) (*http.Response, error) {
 	}
 
 	statusCode := resp.StatusCode
-	if statusCode != http.StatusOK {
+	if statusCode > 200 && statusCode >= 300 {
 		respBody, err := ioutil.ReadAll(resp.Body)
 		if err != nil {
 			return nil, fmt.Errorf("failed to read response[%d]: %w", statusCode, err)
 		}
-		return nil, fmt.Errorf("get request failed: %w", models.NewCfError(url, statusCode, respBody))
+		return nil, fmt.Errorf("%s request failed: %w", req.Method, models.NewCfError(req.RequestURI, statusCode, respBody))
```

Comment:
add Uri into the Cf error output.
Also add add a ToJson (map[string]any) fn for logging.